iroh-content/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+pub mod codecs;
+pub mod content_loader;
+pub mod indexer;
+pub mod types;
+pub mod util;
```

Comment:
no need to make all these `pub` if you have top level reexports

Reply:
good point. did some re-arranging.